presto-docs/src/main/sphinx/migration/from-hive.rst

```diff
@@ -146,3 +146,6 @@ query::
 
 The resulting behavior is equivalent to using `INSERT OVERWRITE
 <https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DML>`_ in Hive.
+
+Overwrite behavior for existing partitions is currently not supported by Presto
```

Comment:
Not sure about the terms "overwrite behavior" ... maybe instead use 

Overwrite operations of existing partitions ...

Reply:
also remove the redundant "currently"